src/model/board_configuration.py

```diff
@@ -180,3 +183,12 @@ def get_scene_by_id(self, scene_id: int) -> Scene | None:
             if scene.scene_id == scene_id:
                 return scene
         return None
+
+    def update_filter_parameter(self, param: update_parameter):
+        for scene in self._scenes:
+            if scene.scene_id == param.scene_id:
+                for filter in scene.filters:
+                    if filter.filter_id == param.filter_id:
+                        # filter.update_parameter(param.parameter_value)
+                        self._broadcaster.update_filter_parameter_direct.emit(param.parameter_value)
```

Comment:
It is totally sufficient to just call `self._broadcaster.update_filter_parameter_direct.emit(param.parameter_value)` as every object that is interested in these updates is notified anyway. These signals are basically a singleton event broker.

Reply:
i thought, than the filter does not need to check it, and does not need to know the protobuf update_parameter class.

problem i went know into: the filter does not know the scene_id, only the name.
should i leave it in two QtCore.Signals, so only the update gets to desired scene?

Another question, the filter in fish dont know its scene_id, right now i send the actual scene_id. There for the question, can you send an receive_gui_updates to fish, for a scene, that is not active? (if not then its not possible to get update for a scene that is not active, and the scene_id is correct corresponding to the scene where the update was transmitted to).

Is the filter ID unique? Or could someone name two cue filter the same, and both will get the same update messages?